source/extensions/omni.isaac.lab/omni/isaac/lab/utils/dict.py

```diff
@@ -253,3 +286,34 @@ def print_dict(val, nesting: int = -4, start: bool = True):
             print(callable_to_string(val))
         else:
             print(val)
+
+
+"""
+Helper functions.
+"""
+
+
+def string_to_slice(s):
```

Comment:
Makes mores sense to move this here?

https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/utils/string.py